docs/en/enterprise-edition/policy-reference/sast-policies/go-policies/sast-policy-255.adoc

```diff
@@ -0,0 +1,79 @@
+== Insecure temporary file creation
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|=== 
+|Prisma Cloud Policy ID 
+| b956c612-c129-477f-849c-64826dc1f0a3
+
+|Checkov ID 
+|CKV3_SAST_255
+
+|Severity
+|MEDIUM
+
+|Subtype
+|Build
+
+|Language
+|Go
+
+|CWEs
+|https://cwe.mitre.org/data/definitions/378.html[CWE-378: Creation of Temporary File With Insecure Permissions]
+
+|OWASP Categories
+|https://owasp.org/Top10/A01_2021-Broken_Access_Control/[A01:2021-Broken Access Control]
+
+|=== 
+
+=== Description
+
+This policy identifies instances where Go applications create files in shared system temporary directories (`/tmp` or `/var/tmp`) without using the `os.CreateTemp` function. Creating temporary files in these directories without proper precautions can allow attackers to create symlinks that point to other files, potentially leading to unintended files being created or overwritten. It is recommended to use `os.CreateTemp` to securely create temporary files.
```

Comment:
This policy checks whether Go applications create files in shared system temporary directories (/tmp or /var/tmp) without using the os.CreateTemp function. Creating temporary files in these directories without proper precautions can potentially enable attackers to exploit symlinks, potentially leading to unintended files being created or overwritten.
@tsmithv11  - remediation moved